package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "clean": "rm -rf dist",
     "compile":
-      "babel src --out-dir dist --copy-files -s inline --extensions '.js,.jsx,.ts,.tsx' --ignore src/**/__tests__,src/**/__stories__",
+      "babel src --out-dir dist --copy-files -s --source-map inline --extensions '.js,.jsx,.ts,.tsx' --ignore src/**/__tests__,src/**/__stories__",
```

Comment:
Isn’t the `-s` option for source-maps already?

Reply:
that `inline` is a typo, which was moved over when I added in `--source-map` which is the command to output files. We were previously embedding sourcemaps into the files via `inline`, increasing weight; need to remove.

Reply:
👍